src/testdir/test_mksession.vim

```diff
@@ -632,8 +634,9 @@ func Test_mksession_sesdir()
   call assert_equal('testdir', fnamemodify(getcwd(), ':t'))
   source Xproj/Xtest_mks2.out
   call assert_equal('Xproj', fnamemodify(getcwd(), ':t'))
-  cd ..
 
+  %bwipe
+  exe 'cd ' . fnameescape(save_cwd)
```

Comment:
`call chdir(save_cwd)` can be also used.

Reply:
`exe 'cd '.save_cwd` is used many times in this file.
Is it appropriate to replace everything?